pkg/storage/unified/sql/data/resource_history_poll.sql

```diff
@@ -8,5 +8,14 @@ SELECT
     {{ .Ident "action" | .Into .Response.Action }}
 
     FROM {{ .Ident "resource_history" }}
-    WHERE {{ .Ident "resource_version" }} > {{ .Arg .SinceResourceVersion }}
+    WHERE 1 = 0
+    {{- range $g, $items := .Since }}
+    {{- range $r, $rv := $items }}
+        OR 
+        ( 
+            "group" = "{{ $g }}" AND "resource" = "{{ $r }}" AND "resource_version" > {{ $rv }} AND 1 = 1
+        )
+    {{- end }}
+    {{- end }}
```

Comment:
This is a neat idea, but I'm not sure how well the database will be able to optimize it

Reply:
Not sure either. The alternative is to poll each resource type independently.

Reply:
Yup, if this drops back to a full table scan we're likely to be better off with per-kind pollers that are able to use the index (and could potentially be started on-demand if we end up with kinds that no-one watches).